test/e2e/e2e_test.go

```diff
@@ -41,7 +43,7 @@ func TestScenarios(t *testing.T) {
 				ArtifactDir:        opts.ArtifactDir,
 				BaseDomain:         opts.BaseDomain,
 			}),
-		"UpgradeControlPlane": scenarios.TestUpgradeControlPlane(ctx,
+		"UpgradeControlPlane [presumbit]": scenarios.TestUpgradeControlPlane(ctx,
```

Comment:
```suggestion
		"UpgradeControlPlane [presubmit]": scenarios.TestUpgradeControlPlane(ctx,
```

Reply:
(I wouldn't care if not fixing this wouldn't mean we have to carry the typo forward to whatever wants to select the tests)

Reply:
ha, good catch!